client/src/avatarRoundel.tsx

```diff
@@ -31,24 +34,19 @@ export const AvatarRoundel = ({
       src={maybeUser?.avatarUrl}
     />
   ) : (
-    <div
-      key={userEmail}
-      title={tooltip}
+    <span
       css={css`
-        background-color: #586293;
-        border-radius: 50%;
-        color: white;
         width: ${size}px;
         height: ${size}px;
+        border-radius: 50%;
+        background-color: ${composer.primary[300]};
+        fill: ${neutral[100]};
         display: flex;
-        justify-content: space-around;
+        justify-content: center;
         align-items: center;
-        font-size: ${size / 2}px;
-        line-height: ${size / 2}px;
       `}
     >
-      {(maybeUser?.firstName || userEmail).charAt(0).toUpperCase()}
-      {maybeUser?.lastName?.charAt(0).toUpperCase()}
-    </div>
+      <SvgPerson size="xsmall" />
```

Comment:
one for @pradasa (as implementation is fine) - sad we're losing the initials, not that many people have a profile pic set and people are familiar with initials from presence etc.

Reply:
Put the initials back as the fallback

![image](https://user-images.githubusercontent.com/10963046/157677214-0e86652b-d0be-49e2-98ce-bf85b0e0eae1.png)